scipy/optimize/_minimize_constrained.py

```diff
@@ -0,0 +1,485 @@
+
+            callback(OptimizeResult state) -> bool
+
+        If callback returns true the algorithm execution is terminated.
+    max_iter : int, optional
+        Maximum number of algorithm iterations.
+    verbose : {0, 1, 2}, optional
+        Level of algorithm's verbosity:
+
+            * 0 (default) : work silently.
+            * 1 : display a termination report.
+            * 2 : display progress during iterations.
+
+    Returns
+    -------
+    `OptimizeResult` with the following fields defined:
+    x : ndarray, shape (n,)
+        Solution found.
+    s : ndarray, shape (n_ineq,)
+        Slack variables at the solution. ``n_ineq`` is the total number
+        of inequality constraints.
+    v : ndarray, shape (n_ineq + n_eq,)
+        Slack variables at the solution. ``n_ineq`` and ``n_eq`` are the
+        total number of equality and inequality constraints.
+    niter : int
+        Total number of iterations.
+    nfev : int
+        Total number of objective function evaluations.
+    ngev : int
+        Total number of objective function gradient evaluations.
+    nhev : int
+        Total number of lagragian Hessian evaluations.
+    ncev : int
+        Total number of constraint evaluations.
+    njev : int
+        Total number of constraint Jacobian matrix evaluations.
+    niter : int
+        Total number of CG iterations.
+    cg_info : Dict
+        Dictionary containing information about the latest CG iteration:
+
+            - niter : Number of iterations.
+            - stop_cond : Reason for CG subproblem termination:
+
+                1. Iteration limit was reached;
+                2. Reached the trust-region boundary;
+                3. Negative curvature detected;
+                4. Tolerance was satisfied.
+
+            - hits_boundary : True if the proposed step is on the boundary
+              of the trust region.
+
+    execution_time : float
+        Total execution time.
+    trust_radius : float
+        Trust radius at the last iteration.
+    penalty : float
+        Penalty function at last iteration.
+    tolerance : float
+        Tolerance for barrier subproblem at the last iteration.
+        Exclusive for ``tr_interior_point``.
+    barrier_parameter : float
+        Barrier parameter at the last iteration. Exclusive for
+        ``tr_interior_point``.
+    status : {0, 1, 2, 3}
+        Termination status:
+
+            * 0 : The maximum number of function evaluations is exceeded.
+            * 1 : `gtol` termination condition is satisfied.
+            * 2 : `xtol` termination condition is satisfied.
+            * 3 : `callback` function requested termination.
+
+    message : str
+        Termination message.
+    method {"eq_constrained_sqp", "tr_interior_point"}
+        Optimization method used.
+    constr_violation : float
+        Constraint violation at last iteration.
+    optimality : float
+        Norm of the lagrangian gradient at last iteration.
```

Comment:
Lagrangian should be capitalized here and elsewhere, as should Hessian (I think you have always capitalized that) and Jacobian.